acceptance/tests/proxy_connection.rb

```diff
@@ -3,6 +3,10 @@
 require 'puppet/acceptance/environment_utils.rb'
 
 test_name 'Connect via proxy' do
+
+  tag 'audit:high',      #
```

Comment:
Not super important. Also spinning up proxy is slowish...

Reply:
Thanks @donoghuc I have changed the risk tage for this test to a 'medium'.